srcpkgs/OpenOrienteering-Mapper/template

```diff
@@ -1,13 +1,14 @@
 # Template file for 'OpenOrienteering-Mapper'
 pkgname=OpenOrienteering-Mapper
 version=0.9.5
-revision=2
+revision=3
 build_style=cmake
 hostmakedepends="doxygen qt5-host-tools qt5-plugin-sqlite qt5-qmake qt5-tools"
 makedepends="qt5-devel clipper-devel proj-devel zlib-devel libgdal-devel
  qt5-tools-devel sqlite-devel cups-devel $(vopt_if location qt5-location-devel)
  $(vopt_if sensors qt5-sensors-devel)
- qt5-plugin-mysql qt5-plugin-odbc qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-tds"
+ qt5-plugin-mysql qt5-plugin-odbc qt5-plugin-pgsql qt5-plugin-sqlite
+ qt5-plugin-tds"
```

Comment:
let's put this change back to how it was, if the only change is the line break

Reply:
It was 85 columns though?

Reply:
it's a soft limit, and that kind of change can make it harder to understand when things were done to a template 🤷‍♀️